authBackEnd/apps/dashboard/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, include
+from .views import DashboardView
+
+urlpatterns = [
+    path('', DashboardView.as_view(), name='dashboard')
+]
+app_name = 'dashboard'
```

Comment:
what is this used for

Reply:
This is the first page that the user will be directed after signing in. in this case, it will be changed to the frontend react app. (still in progress). this dashboard will be deleted, as I created this before the frontend was created to test what I was doing.